src/main/resources/com/dubture/jenkins/digitalocean/SlaveTemplate/config.jelly

```diff
@@ -47,7 +47,15 @@
         </f:entry>
 
         <f:entry title="Idle termination time" field="idleTerminationInMinutes">
-            <f:textbox default="10" />
+            <f:textbox default="60" />
```

Comment:
I'm not so happy with this change. The problem is that you pay for every started hour. If the build took 10 minutes and you're waiting for 60 minutes you already pay two hours. IMHO the only correct solution would be to completely change this feature to only a radio button: 

[ ] keep alive
[ ] shutdown at next full hour

Whilte the first one would never shutdown the container, the second one will keep the container alive till the next full hour is reached and the scheduling algorithm would almost take the container with the longest remaining time to live (but those two things could be two different features). Till we've something like this in place I think the 10 minutes are the saner default choice.